modules/adnowBidAdapter.md

```diff
@@ -0,0 +1,61 @@
+# Overview
+
+```
+Module Name: AdNow Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: support@adnow.com
+```
+
+# Description
+
+AdNow Bidder Adapter for Prebid.js. 
+Banner and Native format are supported.
+Please use ```adnow``` as the bidder code.
+
+# Test Parameters
+```javascript
+    const adUnits = [{
+      code: 'test',
+      mediaTypes: {
+        banner: {
+          sizes: [[300, 250]]
+        }
+      },
+      bids: [{
+        bidder: 'adnow',
+        params: {
+          codeId: 794934,
+          mediaType: 'banner'
+        }
+      }]
+    }, {
+      code: 'test',
+      mediaTypes: {
+        native: {
+          title: {
+            required: true,
```

Comment:
is 'required' a needed property that needs to be passed in or are you telling users that the title property is required? if it's the latter, what value should title be and the information as to what is needed for native can be updated in your markdown file and in your prebid website PR.